homeassistant/helpers/entity_registry.py

```diff
@@ -336,10 +368,36 @@ def _async_update_entity(
                     platform=entity["platform"],
                     name=entity.get("name"),
                     disabled_by=entity.get("disabled_by"),
+                    capabilities=entity.get("capabilities") or {},
```

Comment:
maybe we should not make capabilities optional and use `{}` as default?

Reply:
The reason it's optional is because it is queried EVERY entity update. So otherwise every entity update we're going to create a dict and then copy that dict. I rather have this one check here.